uint8 Initialization=0
uint8 DetectPCB=1
uint8 Move2DetectSOIC=2
uint8 DetectSOIC=3
uint8 Move2PickSyringe=4
uint8 PickSyringe=5
uint8 Move2ReleaseSolderPaste=6
uint8 ApplySolderPaste=7
uint8 Move2DropSyringe=8
uint8 DropSyringe=9
uint8 Move2PickSuction=10
uint8 PickSuction=11
uint8 Move2PickSOIC=12
uint8 PickSOIC=13
uint8 Move2PlaceSOIC=14
uint8 PlaceSOIC=15
uint8 Move2DropSuction=16
uint8 DropSuction=17
uint8 Move2PickHotAirPencil=18
uint8 PickHotAirPencil=19
uint8 Move2SolderPCB=20
uint8 ApplyHotAir=21
uint8 Move2DropHotAirPencil=22
uint8 DropHotAirPencil=23
uint8 ReturnHome=24
int32 actual_state
